src/store/getters/state-getters.ts

```diff
@@ -56,12 +56,12 @@ export const getEntityType = (state: StateIF): CorpTypeCd => {
 
 /** Whether the entity is a Benefit Company. */
 export const isTypeBcomp = (state: StateIF): boolean => {
-  return (state.stateModel.tombstone.entityType === CorpTypeCd.BENEFIT_COMPANY)
+  return (getEntityType(state) === CorpTypeCd.BENEFIT_COMPANY)
 }
 
 /** Whether the entity is a Cooperative. */
 export const isTypeCoop = (state: StateIF): boolean => {
-  return (state.stateModel.tombstone.entityType === CorpTypeCd.COOP)
+  return (getEntityType(state) === CorpTypeCd.COOP)
```

Comment:
I like re-using the simple getters in the store like this 👍

Reply:
Thanks.

I know it might look weird, but it reduces the dependency on the state structure, so we only have to change 1 getter when we update the state object.